Docs/source/running_cpp/parameters.rst

```diff
@@ -589,6 +589,31 @@ Particle initialization
     (the name of an existing positron species must be provided).
     **Implementation of this feature is in progress. It requires `picsar` on the `QED` branch and to compile with QED=TRUE**
 
+* ``<species>.do_resampling`` (`0` or `1`) optional (default `0`)
+    If `1` resampling is performed for this species. This means that the number of macroparticles
+    will be reduced at specific timesteps while preserving the distribution function as much as
+    possible (in particular the weight of the remaining particles will be increased on average).
+    This can be useful in situations with continuous creation of particles (e.g. with ionization
+    or with QED effects). At least one resampling trigger (see below) must be specified to actually
+    perform resampling.
+
+* ``resampling_algorithm.type`` (`string`) optional (default `leveling_thinning`)
+    The algorithm used for resampling. Currently there is only one option, which is already set by
+    default:
+
+    * ``leveling_thinning`` This algorithm is defined in Muraviev et al., arXiv:2006.08593 (2020).
+      The main parameter for this algorithm can be set with ``resampling_algorithm.target_ratio``.
+      It **roughly** corresponds to the ratio between the number of particles before and after
+      resampling. The default value for this parameter is 1.5.
+
+* ``resampling_trigger.intervals`` (`string`) optional (default `0`)
```

Comment:
Potentially also useful for species: e.g e-/e+ need to be resampled more often than ions, etc.? :)